entries/m/mecca.com.au.json

```
@@ -0,0 +1,18 @@
+{
+  "MECCA": {
+    "domain": "mecca.com.au",
+    "additional-domains": [
+      "mecca.com",
```

Mecca.com seems to be the main domain used.

```suggestion
    "domain": "mecca.com",
    "additional-domains": [
      "mecca.com.au",
```